### PR TITLE
Beta2 Release (Fix sniff sleep cast as well)

### PR DESCRIPTION
**Pull request by andrewvc**
Bump to beta2, and fix a potential issue casting values for sniffer sleeps

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 2.0.0-beta2
+ - Massive internal refactor of client handling
+ - Background HTTP sniffing support
+
+## 2.0.0-beta
+ - Only support HTTP Protocol
```
Comment (suyograo) [inline CHANGELOG.md]: Can we be explicit here to say we removed node and transport protocol code and this has been moved to native plugin?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## 2.0.0-beta2
+ - Massive internal refactor of client handling
```
Comment (suyograo) [inline CHANGELOG.md]: Also reduced bulk size to 500

Comment (suyograo) [inline CHANGELOG.md]: Also renamed `host` to `hosts` configuration

Comment (andrewvc) [inline CHANGELOG.md]: Fixed!

File: CHANGELOG.md (hunk 1 above)
Comment (andrewvc) [inline CHANGELOG.md]: Fixed!

**Comment by suyograo** [comment]
@andrewvc I don't see the gemspec changes here?

**Comment by suyograo** [comment]
Also, LGTM. Please squash before you merge

**Pull request closed without merging**